src/main/webapp/script.js

```diff
@@ -314,4 +314,30 @@ function createUserLiElement(user) {
   buttonElement.innerText = user.name;
   liElement.appendChild(buttonElement);
   return liElement;
+}
+
+/** Creates an element that represents a task, including its delete button.
+  * @param {Hashmap} user
```

Comment:
should this also be Element?

Reply:
All of the js stuff to do with users is outdated. I have to rework it. I'll create another PR once I merge this one.

Reply:
But yes. I think it should be a UserData object?